fudge-ratio/pom.xml

```diff
@@ -0,0 +1,51 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.sps</groupId>
+  <artifactId>google-cloud-datastore-todo-list</artifactId>
+  <version>1</version>
+  <packaging>war</packaging>
+
+  <properties>
+    <!-- This project uses Java 8 -->
```

Comment:
[OPINION] I do not know much about app engine but, is Java 1.8 a requirement? I mean why not use Java 13 (or at least Java 10, I miss using var)?

Reply:
I am not sure. This is the version we have been using so far, but idk if it will break if we change it. We can try it for sure